mlrun/model_monitoring/db/tsdb/v3io/stream_graph_steps.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2024 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import mlrun.feature_store.steps
+from mlrun.common.schemas.model_monitoring import (
+    EventFieldType,
+    EventKeyMetrics,
+    EventLiveStats,
+)
+
+
+class ProcessBeforeTSDB(mlrun.feature_store.steps.MapClass):
```

Comment:
We just duplicate the whole class from `stream_processing.py`?

Reply:
no, it's invalid :) I'll remove it from stream_processing.py